crates/swc_ecma_transforms_typescript/src/config.rs

```diff
@@ -18,6 +18,26 @@ pub struct Config {
 
     #[serde(default)]
     pub import_export_assign_config: TsImportExportAssignConfig,
+
+    /// An optimization that inlines TS enum member values within
+    /// the same module. This assumes the enum member values are
+    /// never modified.
+    ///
+    /// Defaults to true.
+    #[serde(default = "true_by_default")]
+    pub ts_enum_is_readonly: bool,
```

Comment:
@magic-akari oh, I've never noticed https://github.com/swc-project/swc/blob/06a2b444993f760fa23d4b4ebc25201c64cde0fa/crates/swc_ecma_transforms_base/src/assumptions.rs#L99-L101

Should it use it instead? I'm not sure what the API would look like here though.

Reply:
It's a JavaScript API for end users.
Since we only change the rust side API, this option is irrelevant.